exonum/src/blockchain/service.rs

```diff
@@ -107,20 +239,31 @@ pub trait Transaction: Message + ExonumJson + 'static {
 /// }
 ///
 /// // Transaction definitions
-/// message! {
-///     struct MyTransaction {
-///         const TYPE = SERVICE_ID;
-///         const ID = MY_TRANSACTION_ID;
-///         // Transaction fields
+/// transactions! {
+///     const SERVICE_ID = SERVICE_ID;
+///
+///     MyTransactions {
```

Comment:
Can you move `const SERVICE_ID` to the` MyTransaction` block? In my opinion it will be more consistent.